### PR TITLE
Bugfix/leaf 571 508 main menu

### PR DESCRIPTION
**Pull request by dastan-rahimi**

### CODE REVIEW

File: LEAF_Request_Portal/templates/menu.tpl [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 <div id="headerMenu_container" style="display: inline-block">
-    <a id="button_showLinks" tabindex="0" class="buttonNorm" alt="Links Dropdown" title="Links">Links</a>
+    <button id="button_showLinks" tabindex="0" class="buttonNorm" alt="Links Dropdown" title="Links" aria-haspopup="true" aria-expanded="false">Links</button>
     <div id="headerMenu_links">
     {include file="menu_links.tpl"}
     </div>
 </div>
 {if $is_admin == true}
-     <a href="./admin/" class="buttonNorm"><img src="../libs/dynicons/?img=applications-system.svg&amp;w=16" alt="Admin Panel" title="Admin Panel" />Admin Panel</a>
+     <button onclick="location.href='./admin/';" class="buttonNorm" alt="Admin Panel" title="Admin Panel" style="padding-bottom: 2px;"><img src="../libs/dynicons/?img=applications-system.svg&amp;w=16" />Admin Panel</button>
```
Comment (nathanws) [inline LEAF_Request_Portal/templates/menu.tpl]: Same issue with `onclick` as above.

File: LEAF_Request_Portal/templates/view_homepage.tpl [hunk 2]
```diff
@@ -1,46 +1,46 @@
 <div class="menu2" style="width: 315px; float: left">
 
-<a href="?a=newform" tabindex="-1">
+<button onclick="location.href='?a=newform';"  href="" tabindex="-1" style="background: none; border: none; text-align: left;">
```
Comment (nathanws) [inline LEAF_Request_Portal/templates/view_homepage.tpl]: Same issue with `onclick` as above.

File: LEAF_Request_Portal/templates/view_homepage.tpl [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
-        <img class="menuIconSmall" src="../libs/dynicons/?img=folder-open.svg&amp;w=76" style="position: relative" alt="Inbox" title="Inbox" />
+        <img class="menuIconSmall" src="../libs/dynicons/?img=folder-open.svg&amp;w=76" style="position: relative"/>
         <span class="menuTextSmall">Inbox</span><br />
         <span class="menuDescSmall">Your inbox is currently empty</span>
     </span>
-</a>
+</button>
 <!--{else}-->
-<a href="?a=inbox" tabindex="-1">
+<button onclick="location.href='?a=inbox';" tabindex="-1" style="background: none; border: none; text-align: left;">
```
Comment (nathanws) [inline LEAF_Request_Portal/templates/view_homepage.tpl]: Same issue with `onclick` as above.

File: LEAF_Request_Portal/templates/view_homepage.tpl [hunk 4]
```diff
@@ -1,46 +1,46 @@
...
+        <img class="menuIconSmall" src="../libs/dynicons/?img=document-open.svg&amp;w=76" style="position: relative"/>
         <span class="menuTextSmall">Inbox</span><br />
         <span class="menuDescSmall">Review and apply actions to active requests</span>
     </span>
-</a>
+</button>
 <!--{/if}-->
 
-<a href="?a=bookmarks" tabindex="-1">
+<button onclick="location.href='?a=bookmarks';" tabindex="-1" style="background: none; border: none; text-align: left;">
```
Comment (nathanws) [inline LEAF_Request_Portal/templates/view_homepage.tpl]: Same issue with `onclick` as above.

File: LEAF_Request_Portal/templates/view_homepage.tpl [hunk 5]
```diff
@@ -1,46 +1,46 @@
...
-        <img class="menuIconSmall" src="../libs/dynicons/?img=bookmark.svg&amp;w=76" style="position: relative" alt="Bookmarks" title="Bookmarks" />
+        <img class="menuIconSmall" src="../libs/dynicons/?img=bookmark.svg&amp;w=76" style="position: relative"/>
         <span class="menuTextSmall">Bookmarks</span><br />
         <span class="menuDescSmall">View saved links to requests</span>
     </span>
-</a>
+</button>
 
-<a href="?a=reports&v=3" tabindex="-1">
+<button onclick="location.href='?a=reports&v=3';" tabindex="-1" style="background: none; border: none; text-align: left;">
```
Comment (nathanws) [inline LEAF_Request_Portal/templates/view_homepage.tpl]: Same issue with `onclick` as above.

**Comment by apgates** [comment]
Tested the changes in both Firefox and IE11. The formatting works in both browsers.

**Pull request merged**